### PR TITLE
GTests renaming: filenames which start with letter C

### PR DESCRIPTION
**Pull request by Vsevolod1983**
GTests renaming according to the document: 
https://github.com/ROCm/MIOpen/wiki/GTest-development

All filenames on letter "c"

Feel free to comment.

### CODE REVIEW

File: test/gtest/cba_infer.cpp [hunk 1]
```diff
@@ -162,21 +162,21 @@ TEST_P(ConvBiasActivInferTestFloatFusionCompileStep, ConvBiasActivAsm1x1UFloat_t
 
 INSTANTIATE_TEST_SUITE_P(
     CBAInferSolverTest,
-    ConvBiasActivInferTestFloatFusionCompileStep,
+    GPU_ConvBiasActivInferFusionCompileStep_FP32,
```
Comment (CAHEK7) [inline test/gtest/cba_infer.cpp]: Forgotten `INSTANTIATE_TEST_SUITE_P`

File: test/gtest/conv_embed_db.cpp [hunk 2]
```diff
@@ -217,10 +217,14 @@ TEST_P(ConvEmbedConfigBFloat16, BFloat16Test_conv_embed_db)
 };
 
 INSTANTIATE_TEST_SUITE_P(ConvEmbedDB,
-                         ConvEmbedConfigFloat,
+                         CPU_ConvEmbedConfig_FP32,
```
Comment (CAHEK7) [inline test/gtest/conv_embed_db.cpp:220]: Forgotten `INSTANTIATE_TEST_SUITE_P`

File: test/gtest/cba_infer.cpp (hunk 1 above)
Comment (Vsevolod1983) [reply test/gtest/cba_infer.cpp]: good catch

**Comment by Vsevolod1983** [comment]
I put CI Passed label because broken windows build is not related to my PR.

**Review by CAHEK7 — APPROVED** [review]

**Review by junliume — APPROVED** [review]

**Pull request merged**